### PR TITLE
merge hannah-book-controller/lexy-import-book-routes to main

### PR DESCRIPTION
**Pull request by hannah68**

### CODE REVIEW

File: src/server/controllers/book.js [hunk 1]
```diff
@@ -2,32 +2,94 @@ const { prisma } = require('../utils/prisma');
...
+		res.status(SERVER_ERROR.NOT_FOUND.CODE).json({ error: SERVER_ERROR.NOT_FOUND.MESSAGE });
+	}
+
+	const book = await prisma.book.findUnique({
+		where: {
+			id: foundGenre.id
+		}
+	})
+
+	res.status(SERVER_SUCCESS.POST_OK.CODE).json({ data: book });
```
Comment (alexamaingard) [inline src/server/controllers/book.js:85]: Maybe we should check for errors here if book is not found

**Review by alexamaingard — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
     ]
   },
   "devDependencies": {
+    "@faker-js/faker": "^6.0.0-beta.0",
```
Comment (Can-Tina) [inline package.json:44]: I assume this is for the user data

File: src/client/App.jsx [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+        <Header/>
+        <Routes>
+          <Route path='/' element={<Home />}/>
+          <Route path='/signup' element={<SignUp />}/>
+          <Route path='/signin' element={<SignIn />}/>
+          <Route path='/match' element={<MatchPage />}/>
+          <Route path='/recommendation' element={<MyRecommendations />}/>
+        </Routes>
+        <Footer/>
+      </div>
```
Comment (Can-Tina) [inline src/client/App.jsx:24]: Perfect, thanks!

**Review by Can-Tina — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (alexamaingard) [reply package.json:44]: Exactly!

**Review by vherus — APPROVED** [review]
Good work with this

There are a few vague commit messages in the history for this branch 
("delete files", "remove function" and "update code"), let's try and keep them meaningful

**Pull request merged**